packages/cli/src/bulkExport.ts

```diff
@@ -0,0 +1,25 @@
+import { Command } from 'commander';
+import { writeFile } from 'fs';
+import { medplum } from '.';
+
+export const bulkExport = new Command('bulk-export');
+
```

Comment:
If someone were to type `medplum bulkExport` what will happen?

Reply:
Good question! I'll create the cli docs as a follow up task. 

`medplum bulk-export` will perform a system level export which exports all resources as per https://www.medplum.com/docs/api/fhir/operations/bulk-fhir#system-level-export. 
[Fhir System Level Export](https://www.medplum.com/docs/api/fhir/operations/bulk-fhir#system-level-export)